tls/s2n_server_hello.c

```diff
@@ -195,16 +205,31 @@ int s2n_server_hello_recv(struct s2n_connection *conn)
     return 0;
 }
 
+static int s2n_hello_retry_required(struct s2n_connection *conn)
```

Comment:
I'm confused to the difference between s2n_hello_retry_required() and s2n_server_requires_retry()

Reply:
These are the same. I've removed this static and use `s2n_server_requires_retry`